templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>Manufacturer List <a style="float: right" href="{% url "taxi:manufacturer-create" %}">+</a>
```

Comment:
Ensure that the `manufacturer-create` URL is correctly defined in your `urls.py` file to avoid any broken links.